src/appkit.cs

```diff
@@ -22295,8 +23007,6 @@ partial interface NSRunningApplication {
 	}
 
 	partial interface NSPasteboard {
-		[Mac (10, 7), Field ("NSPasteboardTypeTextFinderOptions")]
-		NSString PasteboardTypeTextFinderOptions { get; }
 	}
```

Comment:
is the partial `NSPasteboard` interface still needed ?

Reply:
No, I can combine I believe.